lua/telescope/builtin.lua

```diff
@@ -904,4 +904,62 @@ builtin.marks = function(opts)
   }):find()
 end
 
+builtin.tags = function(opts)
+  opts = opts or {}
+
+  if not vim.loop.fs_open('tags', "r", 438) then
+    local answer = vim.fn.input('Tags file does not exists. Do you want to create one? [y/N]')
+    if answer:lower() == 'y' then
+      os.execute('ctags -R --fields=+n')
+    else
+      return
+    end
+  end
```

Comment:
Hmmm I'm not sure I like that this is here. I think instead of doing input, let's just print that people should do do something like this.

If people don't have a tags file I don't think they should try and use `builtin.tags`. Does that make sense?

Reply:
I had this originally and then i took a look at what fzf is doing and they do this. I don't have any hard feelings for both solutions, so up to you.

Reply:
I like `builtin` to not _do_ anything outside of making / doing pickers. I prefer just printing.